README.md

```diff
@@ -25,6 +25,9 @@ MAPS_API_KEY=someKeyString
 
 Pass .env file to flutter run/build via `--dart-define-from-file=.env`
 
+> [!WARNING]
+> In **iOS \*.xcconfig** has some limitations when it comes to variable values it treats the sequence // as a comment delimiter. This means that you won’t be able to specify API_URL for example, https://example.com since everything after // will be ignored. Read about the solution [here](https://nshipster.com/xcconfig/#managing-constants-across-different-environments).
```

Comment:
While the intention is correct, this could be a bit deceiving at first glance.

You can specify the `API_URL` but with `//` treated as a comment delimiter(just as you have said).

The solution is to omit the scheme while specifying the env variable and prepend the scheme in code.

If you could reword this and keep the solution in this doc while referencing some external link also, then we are good to go.